### PR TITLE
Updates to documentation

### PR DESCRIPTION
**Pull request by IanLee1521**
This is to help work on #1. I thought I would start pushing my edits and updates so that I could start getting feedback. I feel as though this could be ready for merging now, but that really I'd like to add some more / make a few more changes.

Couple of major updates / changes.
- I added a setup.py file to allow packaging up this tool and distributing it.
- I created a package on [pypi](https://pypi.python.org/pypi/slack-emoji-search) to allow this to simply be `pip install slack-emoji-search` capable. @micahsaul , I am of course happy to at least add, or even transfer the package to you if you prefer.

I feel like the need to create an api_token.py file (currently what is documented) is a bit clunky... I propose that this might want to change to one of two approaches:

Added a `--api-token=ABCXYZ` commandline argument to allow it to be passed in to the script. And / or add the ability for the scrip to look in a particular location (something like `~/.slack/api_token` ?) that could be where users are then instructed to look for it / to store their key. Thoughts?

As a side note -- Unfortunately, I don't have a team on Slack, so I can't get an API token, and therefore can't fully test things out for you, but just doing source code reading so far, looks pretty good.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+from setuptools import setup
+
+
+setup(
+    name='slack-emoji-search',
+    version='0.0.1',
+    description='Search across Slack for messages with a specific emoji.',
+    author='Micah Saul',
+    author_email='micah.sual@gsa.gov',
```
Comment (wslack) [inline setup.py]: Should be Micah.Saul@gsa.gov

**Comment by IanLee1521** [comment]
@wslack fixed the author_email typo, thanks for spotting that!

File: README.md [hunk 2]
```diff
@@ -2,13 +2,24 @@
 
 A commandline utility to search across Slack for messages that were reacted to with a specific emoji.
 
-## Installation
+## Installation (simple)
 
-Ensure Python and virtualenv are set up. If you've run the [laptop script](https://github.com/18F/laptop), you're all good.
+The simplest way to install this package is to use [pip](https://pip.pypa.io/en/stable/installing/) to install the package from the [Python Package Index](https://https://pypi.python.org/pypi/slack-emoji-search):
```
Comment (brittag) [inline README.md]: https://https://pypi.python.org/pypi/slack-emoji-search has an extra HTTPS in it :)

**Comment by brittag** [comment]
Thanks @IanLee1521, setting up a pypi package is a nice idea - this seems like a helpful improvement overall! And thanks for taking a look through the code. Could it make sense to update the description at https://pypi.python.org/pypi/slack-emoji-search to ask people to refer to this readme, so that the instructions don't get out of sync when people update this one? (I don't know if that's a reasonable practice on pypi.)

**Comment by IanLee1521** [comment]
@brittag -- Thanks for the feedback. I updated the description [on pypi](https://pypi.python.org/pypi/slack-emoji-search) on a one off basis. I have to double check how to do it dynamically.

For what it's worth, the default mode for PyPI is to use the README as the description, snapshotted at that version that is uploaded (so theoretically the Git repo might be newer than the description on PyPI, but that ties the description on PyPI to that version that it is holding. Does that make sense?

File: README.md (hunk 2 above)
Comment (IanLee1521) [inline README.md]: Fixed in 868330a, thanks!

**Comment by micahsaul** [comment]
Looks good to me! Thanks @IanLee1521! I opened #3 to implement your proposal for better token handling.

**Comment by brittag** [comment]
Thanks again @IanLee1521! That makes sense. It could be helpful for us to build a habit of making PyPI packages for utilities; we'll think about how to make that happen.

**Comment by IanLee1521** [comment]
@micahsaul see #5 for a possible crack at what I proposed (just adding the CLI flag as an alternative to creating the `api_token.py` file.

@brittag -- Happy to help! Let me know if you need / want any other input on that. I'm still getting up to speed a bit, but happy to help out (also I'm just very pro the-work-that-18F-is-doing, especially when I put on [one of](https://github.com/llnl) [my work](http://software.llnl.gov) [hats](http://people.llnl.gov/lee1001).

**Pull request merged**